Content.Server/Forensics/Systems/ForensicsSystem.cs

```diff
@@ -64,5 +69,19 @@ private void ApplyEvidence(EntityUid user, EntityUid target)
             if (TryComp<FingerprintComponent>(user, out var fingerprint))
                 component.Fingerprints.Add(fingerprint.Fingerprint ?? "");
         }
+
+        public void TransferForensicsFromDestroyedEntity(EntityUid uid, TransferForensicsOnSpawnBehaviorComponent component, OnSpawnFromSpawnEntitiesBehaviourEvent ev)
+        {
+            if (!EntityManager.TryGetComponent<ForensicsComponent>(uid, out var ownerForensics))
+                return;
+
+            var spawnedForensics = EntityManager.EnsureComponent<ForensicsComponent>(ev.Spawned);
```

Comment:
use proxy methods (TryComp, EnsureComp, etc) and update this with the consideration that owner will always have forensics component after the last review